skrifa/src/outline/cff/mod.rs

```diff
@@ -871,4 +877,21 @@ mod tests {
             Blues::new([-249.0, -239.0].map(Fixed::from_f64).into_iter())
         )
     }
+
+    #[test]
+    fn implied_seac() {
+        let font = FontRef::new(font_test_data::CHARSTRING_PATH_OPS).unwrap();
+        let glyphs = font.outline_glyphs();
+        let gid = GlyphId::new(3);
+        assert_eq!(font.glyph_names().get(gid).unwrap(), "Scaron");
+        let glyph = glyphs.get(gid).unwrap();
+        let mut pen = SvgPen::new();
+        glyph
+            .draw((Size::unscaled(), LocationRef::default()), &mut pen)
+            .unwrap();
+        assert_eq!(
```

Comment:
Can you add a comment explainign what this is proving? E.g. if we didn't have seach X would happen, this result proves that Y happened.

Reply:
Added a comment. Also changed the test to just check the resulting SVG for two `Z`s since we're actually just looking for two contours.